src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -956,6 +956,26 @@ They are available in two places: `beforeSend`/`beforeBreadcrumb` and `eventProc
 
 #### Hints for Events
 
+##### Use-Case: `before-send`
```

Comment:
could we just put all of these on a child page? its not core to usage and is more of "go much deeper on your integration" focus - same bucket as most of advanced usage